activerecord/test/schema/schema.rb

```diff
@@ -992,16 +992,17 @@
   create_table :records, force: true do |t|
   end
 
-  if supports_foreign_keys?
-    # fk_test_has_fk should be before fk_test_has_pk
-    create_table :fk_test_has_fk, force: true do |t|
-      t.integer :fk_id, null: false
-    end
+  drop_table :fk_test_has_fk, if_exists: true
 
-    create_table :fk_test_has_pk, force: true, primary_key: "pk_id" do |t|
-    end
+  create_table :fk_test_has_pk, force: true, primary_key: "pk_id" do |t|
+  end
 
-    add_foreign_key :fk_test_has_fk, :fk_test_has_pk, column: "fk_id", name: "fk_name", primary_key: "pk_id"
+  create_table :fk_test_has_fk do |t|
+    t.integer :fk_id, null: false
+    t.foreign_key :fk_test_has_pk, column: "fk_id", name: "fk_name", primary_key: "pk_id"
+  end
```

Comment:
@sgrif Thank you for explanation, I understand. Actually this works without `if supports_foreign_keys_in_create?`. I removed the if statement.